docs/env.md

```diff
@@ -270,6 +270,14 @@ Example: `https://example.com/api/update-state`.
 - Required: Yes
 - Default: None
 
+#### TF_VAR_update_state_disable_ssl_validation
+
+Whether SSL certificated validation in requests made by the AWS lambdas should be disabled. This allows Django to run on a domain with a self-signed or otherwise invalid certificate in non-production environments.
```

Comment:
s/certificated/certificate?

Actually thought of it: it would also allow to avoid losing video state updates in case our certificate is temporarily invalid? What security do we get by checking this? The state information we convey is nothing sensitive and the shared secret is not transmitted directly... only a valueless signature. But maybe I'm missing something.

Reply:
Not sure. In any case, I'm not comfortable making this change across the board without further discussion.